docs/dotnet/dotnet-http-metrics.md

```diff
@@ -43,15 +43,13 @@ Notes:
 
 ### Metric: `http.client.open_connections`
 
-<!-- semconv metric.dotnet.http.client.open_connections(metric_table) -->
+<!-- Tables in this document are not auto-generated and are intentionally frozen in time. From the .NET perspective this metric and its attributes are stable till the next major version. They are still experimental in the OpenTelemetry. -->
```

Comment:
Should we have this description in the MD file itself, so people visiting the page would also see it?

Reply:
Good point! I wonder if readers of the document would care? They are probably looking for .NET semconv and don't care if it's auto-generated or not. Auto-generation is an optional implementation detail which is important for authors.

So I'd rather not make it visible.

Reply:
Yes, I wouldn't make this visible as well. Our hope is that we re-conciliate this at some point. (🙏 )

Reply:
My point was not about generation itself but rather the fact that there are some deviations between this doc file and other parts of the schema, if it will bring questions later from users. 
But I'm ok with the idea that they should/will be later re-synced again.